phpBB/phpbb/avatar/driver/upload.php

```diff
@@ -13,15 +13,17 @@
 
 namespace phpbb\avatar\driver;
 
+use \phpbb\storage\exception\exception as storage_exception;
```

Comment:
I have used alias in more parts, for filesystem_helper for example, instead of having a filesystem_helper.php